libraries/rush-lib/assets/rush-init/rush.json

```diff
@@ -422,11 +422,29 @@
       "reviewCategory": "production",
 
       /**
-       * A list of local projects that appear as devDependencies for this project, but cannot be
-       * locally linked because it would create a cyclic dependency; instead, the last published
-       * version will be installed in the Common folder.
+       * A list of Rush project names that are allowed to be installed from NPM
+       * instead of locally linking the folder.
```

Comment:
```suggestion
       * A list of Rush project names that are to be installed from NPM
       * instead of linking to the local project.
```

Reply:
Interesting -- I tested two problem cases: 

1. The reference in **package.json** is missing the `workspace:` protocol, but the dependency is NOT listed as `decoupledLocalDependencies`. Running `rush update` reports an error like this:

    `"@rushstack/node-core-library" depends on package "@rushstack/heft-node-rig" (1.9.15) which exists within the workspace but cannot be fulfilled with the specified version range. Either specify a valid version range, or add the package as a cyclic dependency.`

2. The reference in **package.json** does use `workspace:`, but the dependency IS ALSO listed as `decoupledLocalDependencies`. Running `rush update`:

      **No error or warning**
 
From chatting with @iclanton, we're thinking that behavior 1 is correct but needs a better message, whereas behavior 2 is not correct -- `rush update` should either report an error, or silently fix the problem (as in the `useWorkspace=true` upgrade path).  

However these improvements seem beyond the scope of this PR, which was simply renaming a setting.  Thus I think should solve them in a separate PR.

Reply:
CC @dmichon-msft @D4N14L Can you think of any legitimate situation where `cyclicDependencyProjects` would be specified but the `workspace:` protocol is NOT used?